crates/topos-tce-synchronizer/src/lib.rs

```diff
@@ -150,6 +151,14 @@ impl GrpcSynchronizerService for SynchronizerService {
             .unwrap_or(Uuid::new_v4());
         debug!("Received request for checkpoint (request_id: {})", id);
 
+        let limit_per_subnet: usize = max(
```

Comment:
I put `max` for backward compatibility on gRPC, if we receive a message without `limit_per_subnet` it gonna be set to `0`.